src/App.js

```diff
@@ -1,27 +1,9 @@
-import logo from './logo.svg';
-import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit
-          {' '}
-          <code>src/App.js</code>
-          {' '}
-          and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <Calculator />
     </div>
   );
 }
```

Comment:
- [ ] Great job you have imported the Calculator.js in the App.js file, however, The App.js component should be class-based.
![storage1](https://user-images.githubusercontent.com/55635977/133318323-e3a5036c-7db4-43dc-a31e-c470edf9bfb6.png)

Reply:
Hello @Emmyn5600, why do I need to make all my components class-based when linters don't accept them if they are stateless components? 

Check this out: 

![image](https://user-images.githubusercontent.com/85680436/133327389-e0295149-829e-4cd3-8ecc-dc06864bae61.png)

Since best practices tell me to don't disable linters I need to do this: 

![image](https://user-images.githubusercontent.com/85680436/133327908-961351b6-732b-40ab-9836-cb4e174b6eb2.png)

Reply:
Don't mind the previous question, linters let you pass if the state is a empty object!